epilogue_1_how_to_get_there_from_here.asciidoc

```diff
@@ -445,6 +492,8 @@ at a time. If you need to read data to figure out what to do, then consider a
 read-model that returns just the data you need, even if you cheat and build it
 with a repo and domain objects under the hood.
 
+// (EJ3) This makes sense, but what about the pattern where a UoW has multiple repos?
```

Comment:
Not sure I understand this question

Reply:
Since 1 aggregate = 1 repo, and it's not unusual (?) for 1 UoW to have [multiple repos](https://docs.microsoft.com/en-us/aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/implementing-the-repository-and-unit-of-work-patterns-in-an-asp-net-mvc-application#creating-the-unit-of-work-class), this read like a contradiction in my mind.  Reading it a second time, I think there's just some nuance to to this advice.

Reply:
I think maybe it's that we're talking about both a unit of work that needs to be done, like in a use case, and a Unit of Work object that provides access to a consistent snapshot of state and manages commit. Might be worth using the synonym "operation" for all the former.